prereise/gather/demanddata/transportation_electrification/data_helper.py

```diff
@@ -89,3 +90,56 @@ def update_if_weekend(data: pd.DataFrame):
     data.loc[data["Day of Week"].isin(range(2, 7)), "If Weekend"] = 2
 
     return data
+
+
+def get_total_daily_vmt(
+    data: pd.DataFrame,
+    comm_type: int,
+    locationstrategy: List[int],
```

Comment:
`location_strategy` is used in the `smart_charging` function. Why not using the same name for the variable

Reply:
Removed from function since it's not needed anymore